components/dashboard/src/components/SelectWorkspaceClassComponent.tsx

```diff
@@ -80,6 +80,8 @@ export default function SelectWorkspaceClassComponent({
         // if the selected workspace class is not supported, we set an error and ask the user to pick one
         if (selectedWorkspaceClass && !workspaceClasses?.find((c) => c.id === selectedWorkspaceClass)) {
             setError?.(`The workspace class '${selectedWorkspaceClass}' is not supported.`);
+        } else {
+            setError?.(undefined);
```

Comment:
If `g1-standard` is restricted, it will always show an error message because we don't reset it